docs/typography/word-break.md

```diff
@@ -0,0 +1,25 @@
+# Word Break
+
+ZSS contains utility classes for setting values for the `word-break` property.
+
+## Source files
+
+- [utilities/_word-break.scss](../../src/utilities/_word-break.scss)
+
+## Variables
+
+Since the word break CSS properties have a fixed set of values, there is no variable to configure.   
```

Comment:
Als er hier niets over variables te zeggen valt, dan kun je dit kopje ook wel weglaten.

Reply:
In overleg, toch laten staan aangezien het wel gebruikelijk lijkt in de andere docs